### ISSUE TITLE
Manage resolution variation

### ISSUE THREAD

**Issue by JamesZDonline** [JamesZDonline/ArcheoStructures_RV#2]
### Rastervision issue
Currently Rastervision assumes all imagery has the same resolution. This is (mostly) handled on the training side by PyTorch, but on the Predict side of things creates problems.

Assumption: Keeping the scale of image patches consistent is important because the difference between a building structure (house, storage, church, coded 1) and a corral (coded 0) is its size. So changes in scale may make determining correct label difficult.

The Rastervision pipeline currently extracts the patch boundaries for the training data from the geojson files for each scene. This means that all images are on the same scale (~76.8m x76.8m) and higher or lower resolution images just have more or fewer pixels when they're passed to PyTorch. PyTorch can handle that as inputs.

When it comes to prediction, Rastervision cuts the images up into 256x256 pixel patches. For .3m resolution, this results in 76.8m squares for a lower resolution, this results in a patch covering a larger area (scale is changed).

**Comment by JamesZDonline** [comment]
One way to fix this would be to supply rastervision with the imagery's resolution in the scene information and then have the predict tiling function take this information into account for each scene.
